engine.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/dop251/goja"
 	"github.com/dop251/goja_nodejs/console"
 	"github.com/dop251/goja_nodejs/require"
+	esbuild "github.com/evanw/esbuild/pkg/api"
```

Comment:
esbuild itself is a golang package, and so can be invoked directly by our code.

When esbuild is published for the JS ecosystem, it is compiled into either native binaries or wasm, before being wrapped by a bit of javascript code so it can be executed by `node`.